src/lib/hooks.js

```diff
@@ -69,6 +78,19 @@ const api = {
 		updateIframeGuestData({ department });
 	},
 
+	async setBusinessUnitIds(value) {
```

Comment:
```suggestion
	async setBusinessUnit(value) {
```

Why are you expecting multiple business unit Ids? This is not supposed to happen, we'll support only one single BU (ID or Name) as paramenter.

Reply:
Alright :+1: